src/core/router/atoms/showAboutForNewUsers.ts

```diff
@@ -0,0 +1,22 @@
+import { createAtom } from '@reatom/core';
+import { userWasLandedAtom } from '~core/auth/atoms/userWasLanded';
+import history from '../history';
+import { getAbsoluteRoute } from '../routes';
+import { availableRoutesAtom } from './availableRoutes';
+
+export const showAboutForNewUsersAtom = createAtom(
```

Comment:
it looks like this atom is never imported anywhere and never run

Reply:
so that's why this redirection for new user not triggered

Reply:
fixed!